fixtures/showtec/club-par-12-6-rgbwauv.json

```diff
@@ -0,0 +1,187 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Club Par 12/6 RGBWAUV",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Tobias"],
+    "createDate": "2022-10-29",
+    "lastModifyDate": "2022-10-29"
+  },
+  "links": {
+    "manual": [
+      "https://www.highlite.com/de/mwdownloads/download/link/id/17430148/"
+    ],
+    "productPage": [
+      "https://www.highlite.com/de/42651-club-par-12-6-rgbwauv.html"
+    ]
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 12],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
```

Comment:
The manual does indeed say 'Closed', however this doesn't make sense as it would not be possible to have the fixture on solid.

Reply:
Hmm, maybe the light is constantly on for DMX value 255? Or it is indeed a mistake in the manual. I'd keep `Closed` here but add a `helpWanted` message.